draft-ietf-httpbis-no-vary-search.md

```diff
@@ -390,6 +392,36 @@ So, for example, given any non-default value for `No-Vary-Search`, such as `No-V
   <dd><tt>+</tt> and <tt>%20</tt> are both parsed as U+0020 SPACE</dd>
 </dl>
 
+# Caching
+
+If a cache {{HTTP-CACHING}} implements this specification, the presented target URI requirement in {{Section 4 of HTTP-CACHING}} should be replaced with:
```

Comment:
should -> MUST? It feels a bit strong but if you're a cache implementing this spec then I think that this requirement is the whole point, so I guess it makes sense.

Reply:
Strictly speaking I don't think any uppercase (BCP 14) word makes sense because it's not a requirement on the implementation to replace the wording? Just changed it to "is replaced with" to make it sound more descriptive?